src/App.js

```diff
@@ -1,72 +1,78 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
 
+import People from './People';
 class App extends Component {
+  // Classes must have constructors if initial variables are to be set automatically
   constructor() {
-    super()
+    // calls the constructor for the parent (Component) class
+    super();
+    // sets the initail state to include these items
     this.state = {
-      pokemon: [],
-      search: "",
-      selectedPokemon: null
-    }
+      nameInput: '',
+      ageInput: '',
+      people: [
+        { name: 'Alex', age: 21 },
+        { name: 'Ben', age: 18 },
+        { name: 'Abe', age: 31 }
+      ]
+    };
   }
 
-  async componentDidMount() {
-    const res = await fetch('https://pokeapi.co/api/v2/pokemon/')
-    const json = await res.json()
-    this.setState({pokemon: json.results})
-  }
+  addPerson = (event) => {
+    event.preventDefault();
+    const { people } = this.state;
+    // const people = this.state.people;
 
-  onSearchChange = event => {
-    this.setState({search: event.target.value})
-  }
+    const person = {
+      name: this.state.nameInput,
+      age: this.state.ageInput
+    };
+
+    people.push(person);
+    this.setState({ people, nameInput: '', ageInput: '' });
+  };
+
+  deletePerson = (index) => {
+    const { people } = this.state;
+    people.splice(index, 1);
+    this.setState({ people });
+  };
 
-  generateSearchResults = search => {
-    if (search === "") {
-      return []
-    } else {
-      return this.state.pokemon
-        .filter(p => p.name.includes(search))
-        .slice(0, 10)
-    }
+  renderForm() {
+    return (
+      <form onSubmit={this.addPerson}>
+        <label>Name: </label>
+        <input type='text' id='name' value={this.state.nameInput} onChange={(event) => {
+          this.setState({ nameInput: event.target.value });
+        }}
+        />
+        <label>Age: </label>
+        <input type='text' id='age' value={this.state.ageInput} onChange={(event) => {
+          this.setState({ ageInput: event.target.value });
+        }}
+        />
+        <input type='submit' />
+      </form>
+    );
   }
 
-  selectPokemon = async (name) => {
-    const res = 
-      await fetch(
-        `https://pokeapi.co/api/v2/pokemon/${name}/`,
-        {cache: "force-cache"})
-        
-    const json = await res.json() 
-    this.setState({selectedPokemon: json, search: name})
+  // Method to render a div containing the name and age of the youngest person in the state's people array
+  getYoungest() {
+    let youngest = this.state.people.reduce((max, p) => p.age < max.age ? p : max, this.state.people[0]);
```

Comment:
```suggestion
    let youngest = this.state.people.reduce((max, p) => (p.age < max.age) ? p : max, this.state.people[0]);
```

This will probably be a bit easier for someone unfamiliar with the ternary operator to read.